src/hubbleds/stages/stage_one.py

```diff
@@ -219,7 +219,8 @@ def __init__(self, *args, **kwargs):
         sdss_data = self.get_data(SDSS_DATA_LABEL)
         selected = self.get_data(STUDENT_MEASUREMENTS_LABEL).to_dataframe()
         selection_tool = SelectionTool(data=sdss_data, state=self.stage_state,
-                                       selected_data=selected)
+                                       selected_data=selected,
+                                       show_galaxies=self.stage_state.marker_reached("sel_gal1"))
```

Comment:
```suggestion
                                       show_galaxies=self.stage_state.marker_reached("sel_gal1")  & self.stage_state.marker_before("sel_gal3"))
```

This addition makes sure the layer containing all the SDSS galaxies get's turned off. The current version shows the following (arrows show that the PR works and student galaxies show up):
![image](https://user-images.githubusercontent.com/7862929/204636240-45267311-c15d-4fc4-a006-09844c4863c3.png)

With the suggested change, you see the following instead:
<img width="783" alt="image" src="https://user-images.githubusercontent.com/7862929/204636464-7de94fc4-9480-4cc6-8cb5-6f6d06f11174.png">

Reply:
@Carifio24 @patudom – This seems like what I would expect if I went back after going through some later stages. This data is from student 3327. I also thought that at some point this was the behavior and the selected points showed up in red (green+red=yellow). 

But I think w/ or w/o this change, it can be merged

Reply:
I didn't see a place where the galaxy layer got turned off, but if I missed it then I agree it should be included in the initialization logic. (I'm also fine with modifying the behavior to turn it off at some point - the functionality to turn off the SDSS layer is already built in to the selection tool.)

Reply:
I don't _think_ I have seen that logic anywhere either, but I have at times seen just the red dots for the student selections (unless I dreamt it, or maybe it was a bug). But at present there isn't anything that I have seen which turns it off. 

So in that case, I think I am good to just merge what you had w/o the commit suggestion. Sound good 👍🏾 ?

Reply:
Sounds good to me :thumbsup:

Reply:
@johnarban, you are right that in the past, when a student logged out and logged back in, the green catalog dots were not displayed, and without them, the student's galaxies were displayed in red. That was a bug that didn't get fixed before the beta rollout, but I can't reproduce it now.

I will go ahead and merge this for now, with the behavior that the student's galaxies are yellow and the rest of the catalog appears as green. We can decide later if it would make sense for the green catalog dots to not be displayed when the student returns.